Gemfile

```diff
@@ -97,6 +106,9 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+
+  # Make HTTP requests fun again
+  gem 'httparty'
```

Comment:
We don't use Faraday directly, it's just an artifact of using jsonapi-rb, so I'm not sure it would add much to synchronize here. I just wanted something that let me set headers easily.